pkg/versionstream/versionstreamrepo/gitrepo_integration_test.go

```diff
@@ -0,0 +1,70 @@
+// +build integration
+
+package versionstreamrepo_test
+
+import (
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/gits"
+	"github.com/jenkins-x/jx/pkg/versionstream/versionstreamrepo"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	RepoURL    = "https://github.com/jenkins-x/jenkins-x-versions"
+	VersionRef = "v1.0.90"
+	BranchRef  = "master"
+	HEAD       = "HEAD"
```

Comment:
This is probably not the most realistic SHA to checkout, it's would be better to validate a specific SHA. Also I'm worried that the moment that we advance the HEAD then this test will break.

Reply:
It won't break as long we're not asserting that it resolves to a specific tag. If we need to be able to do that, then I agree it makes more sense to use a specific commit.